components/Form/FormSelect/index.tsx

```diff
@@ -107,13 +114,3 @@ export const FormSelect = forwardRef<HTMLButtonElement, MySelectProps<SelectItem
 });
 
 FormSelect.displayName = 'FormSelect';
-
```

Comment:
I removed this. I checked and it seems the only other place this was used was for tests and in the styleguide. I updated those with appropriate alternatives so that we don't have random code/components around like this.